Coffee/src/MainCoffeeMachine.java

```diff
@@ -0,0 +1,127 @@
+import java.util.Scanner;
+
+public class MainCoffeeMachine {
+    public static void main(String[] args) {
```

Comment:
@Arayikkhachatryan jan try to split method on logical parts , as we discussed yesterday